src/platform/platform_winuser.c

```diff
@@ -21,6 +21,8 @@
 uint64_t QuicPlatformPerfFreq;
 uint64_t QuicTotalMemory;
 QUIC_PLATFORM QuicPlatform = { NULL };
+QUIC_PROCESSOR_INFO* QuicProcessorInfo;
```

Comment:
Don't we need to free this in QuicPlatformUninitialize?

Reply:
Already done in the latest commit. :)